kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -229,12 +229,15 @@ def _act_on_advise_error(self, adv_results: dict):
             self._tracking_issue.comment(comment)
 
     def _get_users_with_permission(self) -> typing.List[str]:
-        try:
```

Comment:
seems like we are missing this bit:
```
with cloned_repo(self, self.project.default_branch, depth=1) as repo:
                self.repo = repo
```
please include this bit before try catch block and test it

Reply:
Yeah, that looks to be the issue. Even better, you can use `self.project.get_file_content("OWNERS")` so that only one file gets downloaded.